layouts/default.vue

```diff
@@ -10,41 +10,100 @@
           </NuxtLink>
         </div>
 
-        <!-- Search Bar -->
-        <div class="flex-1 mx-4">
-          <input
-            type="text"
-            placeholder="Search for products..."
-            class="w-full px-4 py-2 rounded-md text-gray-700 outline focus:outline-none focus:ring-2 focus:ring-blue-500"
-          />
-        </div>
-
-        <!-- Icons -->
-        <div class="flex items-center space-x-4">
-          <!-- Wishlist -->
-          <NuxtLink to="/wishlist" class="hover:text-blue-400">

```

Comment:
Please Update from NuxtLink to @click="navigateTo('/anypage')", I'm already import it as global.